<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmPesqAluguel</class>
 <widget class="QMainWindow" name="FrmPesqAluguel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>716</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1124</width>
    <height>716</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lista de Alugueis</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Imagens/Lista_aluguel_preto.png</normaloff>../../Imagens/Lista_aluguel_preto.png</iconset>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;FrmPesqAluguel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.508, y1:0.727, x2:0.505, y2:1, stop:0 rgba(255, 255, 153, 255), stop:1 rgba(255, 255, 255, 255))</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Selecione o Tipo de Pesquisa</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="cbPesquisa">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <item>
           <property name="text">
            <string>Código Aluguel</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Código Cliente</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Código Veículo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nome Cliente</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtPesquisa">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPesquisa">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>50</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
border-radius: 5px;
background-color: rgb(20,20,20);
color:rgb(200,200,200);
font: 87 12pt &quot;Arial&quot;;
}

QPushButton:hover{ 
background-color:rgb(0, 0, 0);
color: rbg(200,200,200)
}

QPushButton:pressed{
background-color:rgb(0,0,30);
color:rbg(200,200,200)
}

</string>
          </property>
          <property name="text">
           <string>Pesquisar</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>D:/Faculdade/projeto integrador 6/Projeto_Integrador_6-main/Projeto_Integrador_6-main/Imagens/Pesquisar.png</normaloff>D:/Faculdade/projeto integrador 6/Projeto_Integrador_6-main/Projeto_Integrador_6-main/Imagens/Pesquisar.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="styleSheet">
         <string notr="true">font: 87 10pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
        </property>
        <column>
         <property name="text">
          <string>Código Aluguel</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nome Cliente</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Código Cliente</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Código Veiculo</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Data Aluguel</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Data Prazo</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Data Devolução</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Valor Aluguel</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Valor Multa</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Km Entrada</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Km Devolução</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Devolução de Veiculo</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Data Devolução</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Multa</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Km de Devolução</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="edtDevolucao">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtMulta">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtSaida">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="btnDevolver">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-radius: 5px;
background-color: rgb(20,20,20);
color:rgb(200,200,200);
font: 87 12pt &quot;Arial&quot;;
}

QPushButton:hover{ 
background-color:rgb(0,0, 0);
color: rbg(200,200,200)
}

QPushButton:pressed{
background-color:rgb(0,0,30);
color:rbg(200,200,200)
}
</string>
        </property>
        <property name="text">
         <string>Devolver</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>D:/Faculdade/projeto integrador 6/Projeto_Integrador_6-main/Projeto_Integrador_6-main/Imagens/Devolver.png</normaloff>D:/Faculdade/projeto integrador 6/Projeto_Integrador_6-main/Projeto_Integrador_6-main/Imagens/Devolver.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
